src/lib/db/project-store.ts

```diff
@@ -238,7 +238,15 @@ class ProjectStore implements IProjectStore {
             .where({
                 project_id: id,
             })
-            .pluck('environment_name');
+            .innerJoin(
+                'environments',
+                'project_environments.environment_name',
+                'environments.name',
+            )
+            .orderBy('environments.sort_order', 'asc')
+            .orderBy('project_environments.environment_name', 'asc')
+            .pluck('project_environments.environment_name')
```

Comment:
I think this is a duplicate line, this seems to cause a bunch of tests to fail on my machine